tests/Rules/UnusedPublicPropertyRule/Fixture/StaticUsedInUnionA.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Rules\UnusedPublicPropertyRule\Fixture;
+namespace TomasVotruba\UnusedPublic\Tests\Rules\UnusedPublicPropertyRule\Fixture;
```

Comment:
these pre-existing tests had wrong namespaces and therefore autolading did not work